wurst/utils/Composer.wurst

```diff
@@ -0,0 +1,54 @@
+package Composer
+
+import Lodash
+import LinkedList
+import ClosureTimers
+
+public interface Composer
+    function execute()
+
+public class ConditionalComposer implements Composer
+    protected Composer Composer
+    protected EmptyFunction<bool> condition
+
+    construct(EmptyFunction<bool> condition, Composer Composer)
+        this.Composer = Composer
+        this.condition = condition
+
+    override function execute()
+        if this.condition.call()
+            this.Composer.execute()
+
+public class StaggeredComposer implements Composer
+    protected Composer Composer
+    protected int amount
+
+    construct(int amount, Composer Composer)
+        this.Composer = Composer
+        this.amount = amount
+
+    override function execute()
+        doPeriodicallyCounted(ANIMATION_PERIOD * 2, this.amount) fishHawkLoop ->

```

Comment:
This variable name looks like you missed it when you factored it out. Default is typically `cb`.